docs/core/install/localized-intellisense.md

```diff
@@ -1,13 +1,11 @@
 ---
 title: Install localized IntelliSense files
 description: Learn how to set up your development machine to use localized IntelliSense files for .NET Core projects in Visual Studio.
-author: mairaw
-ms.author: mairaw
```

Comment:
I think these are already specified in *docfx.json*.

![image](https://user-images.githubusercontent.com/31348972/71589382-4e92fd00-2b2d-11ea-8b21-815a79fa5dcc.png)

![image](https://user-images.githubusercontent.com/31348972/71589391-5d79af80-2b2d-11ea-93e5-c30709172a6c.png)

Reply:
By the way, In the CODEOWNERS files, core/install/** is set for @Thraka. But in the docfx, it's set for @mairaw as it's a sub-directory of core/**.

I 'm not sure if this was intended

Reply:
Yes it's by design. We can override folder owners this way.

Reply:
@mairaw Should I take these over since they're in the install folder? I don't mind owning everything here.

Reply:
Sounds good then. Let's keep the change.